package/python-kiwi-spec-template

```diff
@@ -75,6 +75,23 @@ The KIWI Image System provides an operating system image builder
 for Linux supported hardware platforms as well as for virtualization
 and cloud systems like Xen, KVM, VMware, EC2 and more.
 
+%package -n kiwi-devel
+Summary:        KIWI - Development dependencies
+Group:          %{sysgroup}
+Requires:       gcc
+Requires:       glibc-devel
+Requires:       libffi-devel
+Requires:       trang
+Requires:       libxml2-devel
+Requires:       libxslt-devel
+Requires:       python%{python3_pkgversion}-tox
+Requires:       python%{python3_pkgversion}-%{develsuffix} >= 3.6
+Requires:       python%{python3_pkgversion}-setuptools
+
+%description
+This metapackage installs the necessary dependencies
+to develop KIWI.
```

Comment:
This will confuse people in the spec file. You could just as easily get everything you need to develop kiwi by using `dnf builddep kiwi` or similar.

Reply:
Sorry I can't follow you, what is confusing people ? 

iirc builddep or source-install for zypper installs everything that is needed to build the package, but this is not the same as developing kiwi. For example; trang is used to convert the schema if you develop a schema change. For building the package this is not needed at all. Thus I think a -devel package makes sense to track all that and pull it in

Reply:
`-devel` packages are typically used for building things _on_ the thing, not for developing the thing itself.

Reply:
ok, fine for me and I'm open to any other name that solves the confusion. But the motivation for such a package rather than the former ```helper/install_devel_packages```, you would agree to be a good move, no ?

I'm convinced providing a package to pull the "for kiwi development" packages will be better maintained that this script which was old, outdated and non functional.

So what would be a good name then ?

Reply:
```kiwi-develdeps``` in alignment to our kiwi-systemdeps packages ?

Reply:
I renamed to kiwi-develdeps in alignment with kiwi-systemdeps, thoughts ?